docs/source/commandline/commandline-all.rst

```diff
@@ -1715,45 +2383,51 @@ Scripting Options
 
 **-autoboot_command** *"<command>"*
 
-	Command string to execute after machine boot (in quotes " "). To issue a quote to the emulation, use """ in the string. Using **\\n** will issue a create a new line, issuing what was typed prior as a command.
+    Command string to execute after machine boot (in quotes " "). To issue a
+    quote to the emulation, use """ in the string. Using **\\n** will issue a
+    create a new line, issuing what was typed prior as a command.
 
-	Example:  **-autoboot_command "load """$""",8,1\\n"**
+      Example:  **-autoboot_command "load """$""",8,1\\n"**
```

Comment:
Needs a note that it only works for drivers that support "natural keyboard".

Reply:
Done.